candi/control-plane-kubeadm/patches/kube-controller-manager.yaml.tpl

```diff
@@ -75,7 +75,3 @@ metadata:
   namespace: kube-system
 spec:
   dnsPolicy: ClusterFirstWithHostNet
-# TODO remove after Docker support is dropped
-  securityContext:
-    seccompProfile:
-      type: Unconfined
```

Comment:
Are we sure these parameters are not needed for containerd ?

Reply:
Yes. This is a hack that was merged in #3426.